cli/cmd/cluster.go

```diff
@@ -737,14 +730,30 @@ func createBucketIfNotFound(awsClient *aws.Client, bucket string) error {
 			fmt.Print("\n\n")
 			return err
 		}
-		fmt.Println(" ✓")
 	} else {
-		fmt.Println("￮ using existing s3 bucket:", bucket, "✓")
+		fmt.Print("￮ using existing s3 bucket:", bucket)
 	}
+
+	numTaggingAttempts := 0
```

Comment:
nit: any reason not to use `for i := 0; i++; i < 10` style for loop here?

Reply:
No particular reason, that was just how it occurred to me to implement it. How does this one look?

```go
for i := 0; i < 10; i++ {
	err = awsClient.TagLogGroup(logGroup, tags)
	if err == nil {
		fmt.Println(" ✓")
		return nil
	}
	if !aws.IsErrCode(err, cloudwatchlogs.ErrCodeResourceNotFoundException) {
		break
	}
}

fmt.Print("\n\n")
return err
```